src/extensions/askUserForConsent.ts

```diff
@@ -0,0 +1,100 @@
+import * as _ from "lodash";
+import * as clc from "cli-color";
+import * as marked from "marked";
+import TerminalRenderer = require("marked-terminal");
+
+import { FirebaseError } from "../error";
+import { logPrefix } from "../extensions/extensionsHelper";
+import * as iam from "../gcp/iam";
+import { promptOnce, Question } from "../prompt";
+import * as utils from "../utils";
+
+marked.setOptions({
+  renderer: new TerminalRenderer(),
+});
+
+/**
+ * Returns a string that will be displayed in the prompt to user.
+ * @param {string} role
```

Comment:
In TS files, the type can be omitted in jsdocs

```suggestion
 * @param role
```